test/module/irohad/torii/torii_queries_test.cpp

```diff
@@ -263,7 +276,15 @@ TEST_F(ToriiQueriesTest, FindAccountAssetWhenStatefulInvalid) {
   asset.domain_id = "USA";
   asset.precision = 2;
 
-  EXPECT_CALL(*wsv_query, getAccount("accountA")).WillOnce(Return(account));
+  auto creator = "accountA";
+
+  // TODO: refactor this to use stateful validation mocks
+  EXPECT_CALL(*wsv_query, hasAccountGrantablePermission(
+      creator, account.account_id, can_get_my_acc_ast))
+      .WillOnce(Return(false));
+  EXPECT_CALL(*wsv_query, getAccountRoles(creator))
+      .WillOnce(Return(nonstd::nullopt));
+
   EXPECT_CALL(*wsv_query, getAccountAsset(_, _))
       .Times(0);  // won't be called due to failed stateful validation
```

Comment:
then why it is here

Reply:
To guarantee that this wouldn't be called

Reply:
ok